pkg/controller/vsphere/actuator.go

```diff
@@ -5,21 +5,24 @@ package vsphere
 import (
 	"context"
 	"fmt"
+	"time"
 
 	machinev1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	machinecontroller "github.com/openshift/machine-api-operator/pkg/controller/machine"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/klog"
 	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	scopeFailFmt      = "%s: failed to create scope for machine: %v"
-	reconcilerFailFmt = "%s: reconciler failed to %s machine: %w"
-	createEventAction = "Create"
-	updateEventAction = "Update"
-	deleteEventAction = "Delete"
-	noEventAction     = ""
+	scopeFailFmt        = "%s: failed to create scope for machine: %v"
+	reconcilerFailFmt   = "%s: reconciler failed to %s machine: %w"
+	createEventAction   = "Create"
+	updateEventAction   = "Update"
+	deleteEventAction   = "Delete"
+	noEventAction       = ""
+	requeueAfterSeconds = 20
```

Comment:
Can `time.Duration` be a const? Why not `20 * time.Second` here?

Reply:
I just copied this from elsewhere in our code base.  Prefer to be consistent now, possible clean up all those other instances later.